cmd/utils/flags.go

```diff
@@ -162,20 +162,24 @@ var (
 	}
 	NetworkIdFlag = cli.Uint64Flag{
 		Name:  "networkid",
-		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby)",
+		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby, 5=Görli)",
```

Comment:
I think we can also drop `Morden` from this list. Honestly nobody will even know any more what that was.

Reply:
Ha, true! https://twitter.com/hudsonjameson/status/1205912735734784001